public/stylesheet.css

```
@@ -366,7 +366,7 @@ a {
   margin: 0 calc(-1 * var(--side-offset)) 1.2rem;
   padding: 0.8rem var(--side-offset);
```

To make it pixel-perfect, could you adjust the bottom padding as well?

```suggestion
  padding: 0.8rem var(--side-offset) 0.65rem;
```

---

You're right, it's perfect like that
I applied the suggestion